nms/app/packages/magmalte/app/views/equipment/GatewayDetailMain.js

```diff
@@ -124,13 +131,44 @@ function GatewayCommandDialog(props: CommandProps) {
   );
 }
 
+const AGGREGATION_TITLE = 'Aggregation';
```

Comment:
nit: Could we change this to 'Logs & Events Aggregation'?

Reply:
Tried changing it, it is just too long and wraps around
![Screen Shot 2021-01-09 at 8 55 49 AM](https://user-images.githubusercontent.com/8224854/104103742-97a98100-5258-11eb-8766-35970f1060da.png)